src/plugins/vis_types/pie/public/editor/components/pie.tsx

```diff
@@ -116,6 +129,22 @@ const PieOptions = (props: PieOptionsProps) => {
           value={stateParams.isDonut}
           setValue={setValue}
         />
+        {stateParams.isDonut && (
+          <EuiFormRow label={emptySizeRatioLabel} fullWidth>
+            <EuiButtonGroup
+              isFullWidth
+              name="emptySizeRatio"
+              buttonSize="compressed"
+              legend={emptySizeRatioLabel}
+              options={emptySizeRatioOptions}
+              idSelected={
+                emptySizeRatioOptions.find(({ value }) => value === stateParams.emptySizeRatio)
```

Comment:
nit: I don't like when we need to duplicate any code, what about moving it into render-helper and reuse below.

Reply:
in this case we can write some tests for that